pom.xml

```diff
@@ -806,10 +806,70 @@ under the License.</licenseText>
               <version>7.12.0</version>
             </dependency>
           </dependencies>
+          <executions>
+            <execution>
+              <id>maven-pmd-plugin</id>
+              <goals>
+                <goal>check</goal>
+              </goals>
+              <phase>verify</phase>
+            </execution>
+          </executions>
+        </plugin>
+        <plugin>
+          <groupId>org.openrewrite.maven</groupId>
+          <artifactId>rewrite-maven-plugin</artifactId>
+          <version>6.8.0</version>
+          <configuration>
+            <activeRecipes>
+              <!-- BestPractices -->
+              <!-- <recipe>org.openrewrite.java.recipes.JavaRecipeBestPractices</recipe> -->
+              <!-- <recipe>org.openrewrite.java.recipes.RecipeNullabilityBestPractices</recipe> -->
+              <!-- <recipe>org.openrewrite.java.recipes.RecipeTestingBestPractices</recipe> -->
+              <!-- testing -->
+              <!-- <recipe>org.openrewrite.java.testing.assertj.JUnitToAssertj</recipe> -->
+              <!-- <recipe>org.openrewrite.java.testing.assertj.SimplifyAssertJAssertion</recipe> -->
+              <!-- <recipe>org.openrewrite.java.testing.junit5.JUnit5BestPractices</recipe> -->
+              <!-- staticanalysis -->
+              <!-- <recipe>org.openrewrite.staticanalysis.CodeCleanup</recipe> -->
+              <!-- <recipe>org.openrewrite.staticanalysis.CommonStaticAnalysis</recipe> -->
+              <!-- <recipe>org.openrewrite.staticanalysis.FinalizeMethodArguments</recipe> -->
+              <!-- <recipe>org.openrewrite.staticanalysis.RemoveUnusedLocalVariables</recipe> -->
+              <!-- <recipe>org.openrewrite.staticanalysis.RemoveUnusedPrivateFields</recipe> -->
+              <recipe>org.openrewrite.staticanalysis.RemoveUnusedPrivateMethods</recipe>
+            </activeRecipes>
+            <exclusions>
+              <exclusion>src/mdo/java/WrapperList.java</exclusion>
+              <exclusion>src/mdo/java/WrapperProperties.java</exclusion>
```

Comment:
not working like in:

- https://github.com/pmd/pmd/pull/5739

<img width="1883" alt="image" src="https://github.com/user-attachments/assets/6aba502d-d31e-475c-b305-5d685e32050c" />

Reply:
@timtebeek whats wrong here? its same like in pmd and working there.